docs/docs/guides/integrate/identity-providers/introduction.md

```diff
@@ -0,0 +1,182 @@
+
+
+## Where ZITADEL fits in
+
+ZITADEL positions itself as the central hub or the identity broker in the interaction between your application and various external IdPs. It handles the orchestration of authentication requests and manages the seamless flow of identity verification between your application and the chosen IdP. This is achieved through a process known as federation, where ZITADEL acts as a federated identity provider, integrating IdPs via authentication protocols like OpenID Connect and SAML.
+
+
+## Adding external identity providers to your application
+
+With ZITADEL, you can enhance your application's accessibility by integrating social login IdPs such as Google or other IdPs such as EntraID (formerly known as AzureAD) and Auth0. For organizations with bespoke identity solutions, ZITADEL supports integration with custom-built IdPs that adhere to OpenID Connect or SAML protocols. By default, ZITADEL can serve as the primary user store for your applications. This centralizes user management and simplifies the authentication process, allowing users to sign in with their email and password, or via the external IdPs you've integrated.
+
+
+## The advantages of using ZITADEL
+
+The benefits of integrating ZITADEL for managing external IdPs are multiple:
+
+- **No need for custom authentication code**: Your application only needs to interact with ZITADEL, which handles all communications with external IdPs. This abstraction saves you from the complexity of direct integration with multiple IdPs.
+
+
+- **Unified protocol handling**: ZITADEL abstracts away the specific protocols used by different IdPs. Your application communicates with ZITADEL using a standard protocol (e.g., OpenID Connect), while ZITADEL takes care of the rest.
+
+
+- **Centralized user management**: All user profiles are managed within ZITADEL, allowing for a unified view of user identities regardless of the IdP used for authentication.
+
+
+- **Dynamic profile synchronization**: When users update their profiles on an external IdP, those changes are reflected in ZITADEL at the next login, ensuring that user data remains current.
+
+
+- **Simplified account linking**: ZITADEL can link identities from multiple IdPs to a single user profile, facilitating a cohesive user experience across different authentication methods.
+
+
+- **Enhanced group and policy management**: With support for group synchronization and detailed policy configurations, ZITADEL offers granular control over user access and authentication processes.
```

Comment:
What do you mean with that?
ZITADEL does not allow out of the box group and policy sync.
With actions it would be possible.

Reply:
I removed it altogether.